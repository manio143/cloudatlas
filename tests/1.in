SELECT 2 + 2 AS two_plus_two
