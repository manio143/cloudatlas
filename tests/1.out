/uw: two_plus_two: 4
/pjwstk: two_plus_two: 4
/: two_plus_two: 4
